engine/polyglot-api/src/main/scala/org/enso/polyglot/runtime/Runtime.scala

```diff
@@ -1928,55 +1529,37 @@ object Runtime {
       *
       * @param libraryName the name of the loaded library.
       */
+    @named("deserializeLibrarySuggestions")
     final case class DeserializeLibrarySuggestions(libraryName: LibraryName)
         extends ApiRequest
 
     /** A request to start the background jobs processing. */
+    @named("startBackgroundProcessing")
     final case class StartBackgroundProcessing() extends ApiRequest
 
     /** A notification about started background jobs. */
+    @named("backgroundJobsStartedNotification")
     final case class BackgroundJobsStartedNotification() extends ApiNotification
 
     /** A request to serialize the module.
       *
       * @param module qualified module name
       */
+    @named("serializeModule")
     final case class SerializeModule(module: QualifiedName) extends ApiRequest
 
     /** A request to set the execution environment. */
+    @named("setExecutionEnvironmentRequest")
     final case class SetExecutionEnvironmentRequest(
       contextId: ContextId,
       executionEnvironment: ExecutionEnvironment
     ) extends ApiRequest
 
     /** A response to the set execution environment request. */
+    @named("setExecutionEnvironmentResponse")
     final case class SetExecutionEnvironmentResponse(contextId: ContextId)
         extends ApiResponse
 
-    private lazy val mapper = {
-      val factory = new CBORFactory()
-      val mapper  = new ObjectMapper(factory) with ClassTagExtensions
```

Comment:
If I remember correctly, this was the slow initialization part.

Reply:
yes